brave/brave-core
# Update icons and layout of PIP window

* move "Back to tab" button to the top.
* move controls to the middle.

### video
<img width="475" alt="image" src="https://github.com/brave/brave-core/assets/5474642/ec274591-e501-4080-914a-bb2ddd621c52">

### slide
<img width="468" alt="image" src="https://github.com/brave/brave-core/assets/5474642/38b324b4-2a16-40c4-a7d2-f9615ce81faf">

### video conferencing
<img width="338" alt="image" src="https://github.com/brave/brave-core/assets/5474642/9c0b0678-791e-4a54-83d0-4104fc43b820">



Resolves https://github.com/brave/brave-browser/issues/35259



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
* Go to https://googlechrome.github.io/samples/media-session/index.html
* Enter PIP from "Video", "Video conferencing", "Slide" examples
* PIP window for 3 types of media session should be pictures above.
